pkg/controller/codestatus/logic.go

```diff
@@ -48,8 +50,13 @@ func (c *Controller) CheckCodeStatus(r *http.Request, uuid string) (*database.Ve
 		return nil, http.StatusBadRequest, api.Errorf("missing realm")
 	}
 
-	code, err := c.db.FindVerificationCodeByUUID(uuid)
+	code, err := c.db.FindVerificationCodeByUUID(realm.ID, uuid)
```

Comment:
The pattern we've followed so far has been to hang these off of the `realm` instead of making the realm a parameter.

Reply:
changed